### PR TITLE
Uncomment OADP for GA

### PR DESCRIPTION
**Pull request by apinnick**
No Jira or BZ

Uncommented OADP sections for release.
Added content-type attributes.

Previews:

https://deploy-preview-41186--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/index.html#application-backup-restore-operations-overview (backup/restore overview)

https://deploy-preview-41186--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-features-plugins.html (features, plugins)

https://deploy-preview-41186--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.html (installing overview, plus provider-specific installations)

https://deploy-preview-41186--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html (backup)

https://deploy-preview-41186--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.html (restore)

Peer review needed. No QE required.

### CODE REVIEW

**Comment by apinnick** [comment]
@jeana-redhat Here is the PR for the release on Jan. 31. Could you review just to make sure that the issues you addressed in the BSL/VSL PR have been addressed?

File: backup_and_restore/index.adoc [hunk 1]
```diff
@@ -33,45 +33,44 @@ You can always recover from a disaster situation by xref:../backup_and_restore/c
...
-// * xref:  ../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-applications-restic_backing-up-applications[Restic backups]
-//
-// You restore applications by creating a xref:  ../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore`] CR. You can configure xref:  ../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-hooks_restoring-applications[restore hooks] to run commands in init containers or in the application container during the restore operation.
+OADP backs up and restores Kubernetes resources and internal images, at the granularity of a namespace, by using link:https://velero.io/[Velero 1.7]. OADP backs up and restores persistent volumes (PVs) by using snapshots or Restic. For details, see xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features].
+
+.OADP requirements
```
Comment (jeana-redhat) [inline backup_and_restore/index.adoc:38]: This and `Backing up and restoring applications` below should probably be `==` headings, with IDs and `[discrete]` if you do not want them to appear in the TOC. Among other things, this allows people to link to the section if needed. 

I don't see this explicitly prohibited in the guidelines, but based on talking to another reviewer we're pretty sure the `.Heading` format should be reserved for the types of examples given in the guidelines (e.g. `.Prerequisites`).

**Review by jeana-redhat — APPROVED** [review]
Looks good! One formatting comment, but it's very minor. I think you also need to add the new `:_content-type: ASSEMBLY` line right at the beginning of backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc

**Pull request merged**